src/Entity/ResearchGroup.php

```diff
@@ -335,18 +336,35 @@ public function serializeDatasets()
                 'id' => $dataset->getId(),
                 'title' => $dataset->getTitle(),
                 'udi' => $dataset->getUdi(),
-                'citation' => $dataset->getCitation(),
+                'availabilityStatus' => $dataset->getAvailabilityStatus(),
+                'doi' => array(
+                    'doi' => ($dataset->getDoi()) ? $dataset->getDoi()->getDoi() : ''
+                ),
+                'acceptedDate' => ($dataset->getAcceptedDate()) ? $dataset->getAcceptedDate()->format('Y-m-d') : ''
             );
             if (null !== $dataset->getDif()) {
                 $datasetArray['dif'] = array(
                     'id' => $dataset->getDif()->getId(),
                     'status' => $dataset->getDif()->getStatus(),
                     'title' => $dataset->getDif()->getTitle(),
-                    'citation' => $dataset->getCitation(),
                 );
             } else {
                 $datasetArray['dif'] = null;
             }
+            if ($dataset->hasDatasetSubmission()) {
+                $datasetArray['datasetSubmission'] = array(
+                    'authors' =>  $dataset->getDatasetSubmission()->getAuthors(),
+                    'themeKeywords' => $dataset->getDatasetSubmission()->getThemeKeywords()
+                );
+                $datasetArray['fileFormat'] = $dataset->getDatasetSubmission()->getDistributionFormatName();
+                if ($dataset->getDatasetSubmission()->isDatasetFileInColdStorage()) {
+                    $datasetArray['fileSize'] = TwigExtentions::formatBytes($dataset->getDatasetSubmission()->getDatasetFileColdStorageArchiveSize(), 2);
```

Comment:
We should probably move formatBytes to a utility.

Reply:
We certainly could, but it kind of makes sense out on the edge, just like applying any other localization.